package/gluon-core/luasrc/lib/gluon/upgrade/210-interface-mesh

```diff
@@ -49,6 +50,7 @@ if #mesh_interfaces_other > 0 then
 		igmp_snooping = false,
 		proto = 'gluon_wired',
 		index = 4,
+		hop_penalty = 0,
```

Comment:
All values set in this section are overwritten on every upgrade/reconfigure. Let's not set the value in the section at all by default.

Reply:
This is now correctly read from gluon config using:

`hop_penalty = uci:get('gluon', 'batadv_hop_penalty', name) or 0`